src/i18n.ts

```diff
@@ -1,55 +1,72 @@
 import LanguageDetector from "i18next-browser-languagedetector";
 
 export const languages = [
-  { lng: "en", label: "English", data: require("./locales/en.json") },
-  { lng: "bg-BG", label: "Български", data: require("./locales/bg-BG.json") },
-  { lng: "de-DE", label: "Deutsch", data: require("./locales/de-DE.json") },
-  { lng: "es-ES", label: "Español", data: require("./locales/es-ES.json") },
-  { lng: "ca-ES", label: "Catalan", data: require("./locales/ca-ES.json") },
-  { lng: "el-GR", label: "Ελληνικά", data: require("./locales/el-GR.json") },
-  { lng: "fr-FR", label: "Français", data: require("./locales/fr-FR.json") },
+  { lng: "en", label: "English", data: "en.json" },
```

Comment:
Some thoughts:

- `code` is a better name for the language code than `lng`
- `data` is now a misnomer since it doesn’t contain i18n data. Since it is just `lng + .json"` could it be derived at runtime so we only need the `code` and `label`?

Reply:
- `lng` is what the i18n library uses, and it's used through the app.
- While we can derive path from `lng`, maybe it's best to keep the path for more expressiveness. But I don't have a preference, so we can drop it.